### DISCUSSION TITLE
nextauth.js prisma(mysql) sendgrid

### DISCUSSION THREAD

**Discussion by oscarayala431** [prisma/prisma#5412] [Q&A]
Hi, I don't know if anyone has encountered this problem. You are performing a login with nextauth.js with the mail provider. According to the errors that can occur is "get_user_by_email_error", which indicates an error when interacting with the database, but I can't find where I am failing. I am using sendGrid as mail provider. thanks for the help.

![image](https://user-images.githubusercontent.com/52139082/106535711-738a3a00-64bc-11eb-8f98-010480e19a4f.png)

**Comment by pantharshit00** [comment]
See: https://github.com/prisma/prisma/discussions/5269#discussioncomment-307773

Upgrade to canary version of nextauth which has a fix for this.

**Reply by williamluke4** [reply]
I believe that they have now released a fix to stable

**Reply by oscarayala431** [reply]
one more doubt, uh since when executing the prisma init command it generates an .env file and nextjs uses a .env.local file I should use only one .env file right? or it is wrong. thank you both for your help, I appreciate it.

**Reply by pantharshit00** [reply]
Yep, just use `.env`. It should work with everything.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
